opencti-platform/opencti-graphql/tests/02-integration/01-database/elasticSearch-test.js

```diff
@@ -531,7 +531,7 @@ describe('Elasticsearch pagination', () => {
     let data = await elPaginate(testContext, ADMIN_USER, READ_RELATIONSHIPS_INDICES);
     expect(data).not.toBeNull();
     const groupByIndices = R.groupBy((e) => e.node._index, data.edges);
-    expect(groupByIndices[`${ES_INDEX_PREFIX}_internal_relationships-000001`].length).toEqual(14);
+    expect(groupByIndices[`${ES_INDEX_PREFIX}_internal_relationships-000001`].length).toEqual(13);
```

Comment:
Why we have a decrement here ?

Reply:
13 instead of 14 because no more relation between user and role --> the relationship 'has-role' is not counted anymore

Reply:
(I have checked that by printing the 14 previous inputs and the 13 new ones)

Reply:
that's much clearer, thank you!